lnrpc/devrpc/dev.proto

```diff
@@ -30,7 +30,36 @@ service Dev {
     used for development.
     */
     rpc ImportGraph (lnrpc.ChannelGraph) returns (ImportGraphResponse);
+
+    /*
+    AddAliases creates a set of new alias mappings. The list of successfully
+    created mappings is returned. This is only a locally stored alias, and will
+    not be communicated to the channel peer via any message.
```

Comment:
If these aren't communicated to the peer, then what does the peer do when it receives an htlc with one of our alias SCIDs (that it doesn't attribute to us because we never told them about it) as the next hop?